frontend/class-frontend.php

```diff
@@ -1339,6 +1339,7 @@ public function page_redirect() {
 
 			$redir = $this->get_seo_meta_value( 'redirect', $post->ID );
 			if ( $redir !== '' ) {
+				header( 'X-Redirect-By: Yoast SEO' );
```

Comment:
In WordPress 5.0.0 (which isn't released yet) we can pass a third argument for setting this header. 

https://core.trac.wordpress.org/browser/trunk/src/wp-includes/pluggable.php#L1205

This header might be overwritten by the default one.

Reply:
Good call, let's make sure we fix that when 5.0 comes out :D